### PR TITLE
Day5

### PR DESCRIPTION
**Pull request by rushikesh-thinkitive**

### CODE REVIEW

File: Mongoose/Project 1/app.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+const { json } = require('express');
+const mongoose = require('mongoose');
+
+
+const connectDb = ()=>{
+        return mongoose.connect("mongodb://localhost:27017/Temp")
+}
+
+const readData = async ()=>{
+    let DB = await connectDb();
```
Comment (imtiyazthink) [inline Mongoose/Project 1/app.js]: No need to connect to db each time here
It should happen on start of the application and schema creation as well, it should happen only once

File: Mongoose/Project 1/app.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    let studentSchema = new mongoose.Schema({
+        name : { type:String, required:true, trim:true},
+        salary : { type:mongoose.Decimal128, required:true},
+        age : { type:Number,required:true, min:18, max:50},
+    });
+
+    //Compiling Schema
+    const studentModel = mongoose.model('Student',studentSchema)
+    const studentDoc = studentModel({
+        name:"RK",
```
Comment (imtiyazthink) [inline Mongoose/Project 1/app.js]: Should get the details from the request body, instead of hardcoding them

**Review by imtiyazthink — CHANGES_REQUESTED** [review]

File: Mongoose/Project 1/app.js (hunk 1 above)
Comment (rushikesh-thinkitive) [reply Mongoose/Project 1/app.js]: Updated

File: Mongoose/Project 1/app.js (hunk 2 above)
Comment (rushikesh-thinkitive) [reply Mongoose/Project 1/app.js]: Updated

**Review by imtiyazthink — APPROVED** [review]

**Pull request merged**